server/modules/playlist_driver.js

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
new file, so 2017?

Reply:
I don't know how we should handle dates here... How about I update them all in some big pull request in a bit?

Reply:
sure.  I'll see if i can dig up the official policy.